ChangeLog

```diff
@@ -3,12 +3,106 @@ Changelog
 
 ## v2.4.0
 
-- Enum: uniq is modified to use structural comparison internally, instead
-    of previously used physical comparison, which was most likely a bug.
-    The old behavior is provided by the new Enum.uniqq function.
-    Enum.uniq_by was added for additional convenience.
+- ignore deprecation warnings
+  (Gabriel Scherer)
+- BatBitSet: use Bytes instead of String
+  (Gabriel Scherer)
+- rm qtest-generated file all_tests.ml upon 'make clean'
+  (Francois Berenger)
+- corrected many warnings
+  (Francois Berenger)
+- regression tests for issue #609
```

Comment:
skip -- the fix entry "BatHashtbl: ..." is enough. When a fix was reported in the issue list, could you maybe add the PR number?